docs/Versions.asciidoc

```diff
@@ -16,7 +16,7 @@ release-state can be: released | prerelease | unreleased
 :painless:        https://www.elastic.co/guide/en/elasticsearch/painless/master
 :plugins:         https://www.elastic.co/guide/en/elasticsearch/plugins/{branch}
 :javaclient:      https://www.elastic.co/guide/en/elasticsearch/client/java-api/{branch}
-:xpack:           https://www.elastic.co/guide/en/x-pack/{branch}
+:xpack-ref:           https://www.elastic.co/guide/en/x-pack/{branch}
```

Comment:
Please line up the URL with the ones before and after.  You don't use tabs by any chance, do you?  We have a build check that will cause tabs to fail the build.